### DISCUSSION TITLE
Washer program buttons

### DISCUSSION THREAD

**Discussion by englishfox90** [nVuln/homebridge-lg-thinq#242] [General]
I've got a wash and dryer and there not always consistent in working so I'm testing out v1 vs v2. But is it possible to remove the additional button tile that gets added? It doesn't do anything for me, and I can't see how to remove them. 

Thanks

**Comment by nVuln** [comment]
if that button removed, washer tile will be disappear too, that is ios 16 issue, I still find a solution
